packages/wrangler/src/api/form_data.ts

```diff
@@ -53,6 +53,7 @@ export interface WorkerMetadata {
         service: string;
         environment: string;
       }
+    | object
```

Comment:
I think this could be a better type. Don't they all have a name/type at least? Otherwise this will act as a catchall and weaken our types. In fact, I'd rather this not be here and we add a `// @ts-expect-error` where we add to the form.

Reply:
At minimum, there should always be a `name: string` and `type: string`

Reply:
I opted to remove the type